docs/reference/cluster/state.asciidoc

```diff
@@ -38,47 +42,46 @@ that the latest cluster state is returned. For debugging purposes, you can
 retrieve the cluster state local to a particular node by adding `local=true` to
 the query string.
 
-[float]
-==== Response Filters
+[[cluster-state-api-path-params]]
+==== {api-path-parms-title}
 
 The cluster state contains information about all the indices in the cluster,
 including their mappings, as well as templates and other metadata. This means it
 can sometimes be quite large. To avoid the need to process all this information
 you can request only the part of the cluster state that you need:
 
-[source,js]
---------------------------------------------------
-GET /_cluster/state/{metrics}
-GET /_cluster/state/{metrics}/{indices}
---------------------------------------------------
-// CONSOLE
-
-`{metrics}` is a comma-separated list of the following options.
-
-`version`::
-    Shows the cluster state version.
-
-`master_node`::
-    Shows the elected `master_node` part of the response
-
-`nodes`::
-    Shows the `nodes` part of the response
-
-`routing_table`::
-    Shows the `routing_table` part of the response. If you supply a comma
-    separated list of indices, the returned output will only contain the routing
-    table for these indices.
-
-`metadata`::
-    Shows the `metadata` part of the response. If you supply a comma separated
-    list of indices, the returned output will only contain metadata for these
-    indices.
-
-`blocks`::
-    Shows the `blocks` part of the response.
-
-`_all`::
-    Shows all metrics.
+include::{docdir}/rest-api/common-parms.asciidoc[tag=index]
```

Comment:
Can we switch the other of the `{metrics}` and `{index}` parameters?

This will present them in the order they're presented in the path.